### ISSUE TITLE
Cant hold key when selecting shaders

### ISSUE THREAD

**Issue by cyberbug-2077** [FransBouma/ShaderToggler#22]
I cant hold key anymore when selecting trough shaders, must press each time

**Comment by FransBouma** [comment]
Yeah I think it's a reshade thing that it doesn't repeat it anymore. I have noticed it too (and my keyboard isn't happy about it).

Will see if I can find a way around it.

**Comment by cyberbug-2077** [comment]
Ah so its the reshade, will use the earlier version for now
